controllers/spec/common.go

```diff
@@ -341,7 +341,7 @@ func MakeStatefulSetSpec(replicas *int32, container *corev1.Container,
 func MakePodTemplate(container *corev1.Container, volumes []corev1.Volume,
 	labels map[string]string, policy v1alpha1.PodPolicy,
 	downloaderContainer *corev1.Container) *corev1.PodTemplateSpec {
-	podSecurityContext := getDefaultRunnerPodSecurityContext(DefaultRunnerUserID, DefaultRunnerGroupID, false)
+	podSecurityContext := getDefaultRunnerPodSecurityContext(DefaultRunnerUserID, DefaultRunnerGroupID, getEnvOrDefault("RUN_AS_NON_ROOT", "true"))
```

Comment:
Noticed that the default value is changed from `false` to `true` here, maybe it will break the backward compatibility.

Reply:
okay, let's keep the default value as `false` and solve that via system env